rpcd/playbooks/roles/rpc_support/tasks/main.yml

```diff
@@ -95,3 +95,4 @@
 
 - include: configure_unattended_upgrades.yml
 
+- include: create_default_flavors.yml
```

Comment:
Good suggestion, although @d34dh0r53 and @prometheanfire suggested to avoid using when as it is expensive.